apps/root/src/common/components/token-amount-input/index.tsx

```diff
@@ -147,17 +147,17 @@ const TokenAmountInput = ({
                 sx={{ ...buildTypographyVariant(mode).h3, fontWeight: '700', color: 'inherit', textAlign: 'right' }}
               />
             </FormControl>
-            <ContainerBox>
+            <ContainerBox gap={1}>
               <Typography variant="body">≈{` $${tokenAmount.amountInUSD || '0'}`}</Typography>
               {priceImpact && !isNaN(Number(priceImpact)) && isFinite(Number(priceImpact)) && (
                 <Typography
                   variant="body"
                   color={
                     // eslint-disable-next-line no-nested-ternary
                     Number(priceImpact) < -2.5
-                      ? colors[mode].semantic.error.primary
+                      ? colors[mode].semantic.error.darker
                       : Number(priceImpact) > 0
-                      ? colors[mode].semantic.success.primary
+                      ? colors[mode].semantic.success.darker
                       : baseColors.disabledText
```

Comment:
I believe disabledText should be changed to a theme-based color
<img width="155" alt="Screenshot 2024-03-04 at 18 09 11" src="https://github.com/Mean-Finance/dca-fe/assets/144699868/913f9f3a-1fc3-44b6-86b9-b15e6ab31d79">

Reply:
oh good point